app/components/views/TicketsPage/PurchaseTab/LEGACY_PurchasePage/index.js

```diff
@@ -8,6 +8,7 @@ import { ExternalLink } from "shared";
 import { EXTERNALREQUEST_STAKEPOOL_LISTING } from "main_dev/externalRequests";
```

Comment:
Can we rename this `index.js` to something more meaningful ? and use `index.js` to export it properly.